containers/Makefile

```diff
@@ -68,6 +68,10 @@ postgresql-clean:
 concent-api-clean:
 	docker rmi $(IMAGE_PREFIX)concent-api:$(IMAGE_VERSION) --force
 
+
+build/virtualenv/: $(CONCENT_DIR)
+	package-builder/build-virtualenv.sh "$(CONCENT_DIR)" "build"
```

Comment:
This should depend only on `concent_api/requirements.lock`. Not the whole `CONCENT_DIR`.

Reply:
Pass the path to requirements not `CONCENT_DIR` in a parameter.